docs/source/start-software.rst

```diff
@@ -62,24 +63,41 @@ to do the following (and more):
 * Create algorithms and computations
 * Create and manage time series in database
 
+.. image:: ./media/start/software/im-00-swiss-army-knife.JPG
+   :alt: opendcs swiss army knife
+   :width: 800
+
+OpenDCS is designed to operate with a database.  A fast majority 
+of the documentation here assumes that a database is already set 
+up an running.  Below is a high level schematic to conveys how
+a typical office has OpenDCS set up.  The set-up and install 
+directions below focus on a Windows PC set-up.  Further documatation
+about server set-up and how to set up an OpenHydro database can
```

Comment:
This may need a call out that within the old docs and actual code the OpenHydro database is stilled called "OpenTSDB" or "OTSDB"